op-node/rollup/derive/span_batch.go

```diff
@@ -58,6 +58,9 @@ func (b *RawSpanBatch) GetBatchType() int {
 
 // decodeOriginBits parses data into bp.originBits
 func (bp *spanBatchPayload) decodeOriginBits(r *bytes.Reader) error {
+	if bp.blockCount > MaxSpanBatchElementCount {
+		return ErrTooBigSpanBatchSize
+	}
```

Comment:
is this (and other limits) tested?

Reply:
I only added these when these limits where being tested. IMO the tests for these limits are not great because it is modifying the internal state of `*spanBatchPayload` to be invalid and then calling this function.